Assets/MixedRealityToolkit/Inspectors/ServiceInspectors/ServiceFacadeInspector.cs

```diff
@@ -0,0 +1,405 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
+
+using Microsoft.MixedReality.Toolkit.Boundary;
+using Microsoft.MixedReality.Toolkit.Diagnostics;
+using Microsoft.MixedReality.Toolkit.Editor;
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.SpatialAwareness;
+using Microsoft.MixedReality.Toolkit.Utilities.Editor;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Facades
+{
+    [CustomEditor(typeof(ServiceFacade))]
+    [InitializeOnLoad]
+    public class ServiceFacadeEditor : UnityEditor.Editor
+    {
+        static ServiceFacadeEditor()
+        {
+            // Register this on startup so we can update whether a facade inspector is updated or not
+            SceneView.onSceneGUIDelegate += DrawSceneGUI;
+        }
+
+        private static List<string> dataProviderList = new List<string>();
+        private static Dictionary<Type, Type> inspectorTypeLookup = new Dictionary<Type, Type>();
+        private static Dictionary<Type, IMixedRealityServiceInspector> inspectorInstanceLookup = new Dictionary<Type, IMixedRealityServiceInspector>();
+        private static bool initializedServiceInspectorLookup = false;
+
+        Color proHeaderColor = (Color)new Color32(56, 56, 56, 255);
+        Color defaultHeaderColor = (Color)new Color32(194, 194, 194, 255);
+
+        const int headerXOffset = 48;
+        const int docLinkWidth = 175;
+
+        [SerializeField]
+        private Texture2D logoLightTheme = null;
+
+        [SerializeField]
+        private Texture2D logoDarkTheme = null;
+
+        protected virtual void Awake()
+        {
+            string assetPath = $"{MixedRealityEditorSettings.MixedRealityToolkit_RelativeFolderPath}/StandardAssets/Textures";
```

Comment:
>  string assetPath = $"{MixedRealityEditorSettings.MixedRealityToolkit_RelativeFolderPath}/StandardAssets/Textures"; [](start = 10, length = 116)

This line is throwing a warning because it's using a deprecated API. Please use the MixedRealityToolkitFiles helper.

I THINK you wanna do something like

MixedRealityToolkitFiles.MRTKDirectories.First(), though Andrei would be more authoritative than I am.

The reason this was done was to make it so that you could import our code/binaries into not the root assets location.